webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -110,7 +111,7 @@ const isLinkCell = (cell: GridCell): cell is LinkCell => {
 
 export const GlideTable: React.FC<GlideTableProps> = ({
   data,
-  fetchExperiments,
+  dataTotal = 0,
```

Comment:
nit: `dataTotal` isn't optional, so I don't think the default assignment here does anything

Reply:
I just wanted to be 110% sure lol